### ISSUE TITLE
Cannot start xlights

### ISSUE THREAD

**Issue by bossier17** [smeighan/xLights#2168]
I was changing the directory location and xlights crashed.  Now it won't start - when I open xlights it opens the box asking for email address to report bugs and then cursor starts to spin and lights freezes.  Have to force close.  Can't even open it to get logs or anything.  

Tried restarting computer and reinstalling xlights and still no luck.   Mac 10.15.6. Not sure what xlights version as I can’t open it now but downloaded off App Store.  

NOTE: IF YOU DO NOT ATTACH A SEQUENCE THAT DEMONSTRATES YOUR PROBLEM THEN THERE IS A HIGH PROBABILITY YOUR ISSUE WILL JUST BE CLOSED AND NOT INVESTIGATED.

**Describe the bug**

**To Reproduce**
(and/or link to a short video showing the problem with audio track describing what you are doing)

**Expected behavior**
A clear and concise description of what you expected to happen and why.

**Screenshots**

**Versions (please complete the following information):**
 - OS: [e.g. Windows 10, Mac 10.14.5, etc...]
 - xLights version [e.g. 2019.37]

**Additional context**

**Attachments**
- If associated with a crash please attach/link to a dropbox containing the crash log that was produced
- If it crashed but no crash log was produced please use Tools Menu/Package Logs and attach/link to a dropbox containing the resulting zip file
- If the error relates to rendering an effect or a crash while sequencing please use Tools Menu/Package Sequence and attach/link to a dropbox containing the resulting zip file

If you would prefer not to post your sequences publically please say so and be ready to PM them to one of the developers when asked.

**Comment by computergeek1507** [comment]
If you open cmd.exe on windows and run the "xlights -w" it will wipe your settings.(it doesn't delete your show folder) If you could zip and upload your show folder, we could look into it more.

![image](https://user-images.githubusercontent.com/2293065/90259797-9c3c4b80-de18-11ea-9850-de2f1c206c96.png)

**Comment by bossier17** [comment]
I have a Mac so couldn't do what you asked.  I think I solved it though - it crashed when trying to change the show directory to the one where I had stored LithgowLights "thunder" that I had downloaded from Xlights.   I went ahead and moved that folder, and the next time I tried to start Xlights it said the show directory could not be found, had be specify a new one, and then it worked fine.   I haven't tried to use that show directory again.  For some reason trying to open that show directory cause Xlights to crash, and so it would crash on startup since it was trying to open that one right away.  thanks for the help!
